components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -131,6 +131,14 @@ public abstract void updateApplication(ServiceProvider serviceProvider, String t
     public abstract void deleteApplication(String applicationName, String tenantDomain, String username)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Delete Applications by tenant id.
+     *
+     * @param tenantId Id of the tenant
+     * @throws IdentityApplicationManagementException
+     */
+    public abstract void deleteApplicationsByTenantId(int tenantId) throws IdentityApplicationManagementException;
```

Comment:
rename to deleteApplications(int tenantId)

Reply:
👍